### PR TITLE
Change the strategy to classify packages by only looking at the lib folder

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

**Comment by javiercn** [comment]
@pranavkm @natemcmaster

File: src/PackageClassifier/PackageSourcesCache.cs [hunk 1]
```diff
@@ -44,10 +45,16 @@ private PackageInformation ReadPackageInformation(FileSystemInfo file)
...
                     identity.Version.ToString(),
-                    reader.GetSupportedFrameworks().Select(fx => fx.DotNetFrameworkName));
+                    GetSupportedFrameworks(reader));
             }
         }
 
+        private IEnumerable<string> GetSupportedFrameworks(PackageArchiveReader reader)
+        {
+            // To array is required here to force the enumeration.
+            return reader.GetLibItems().Select(fsg => fsg.TargetFramework.DotNetFrameworkName).ToArray();
```
Comment (pranavkm) [inline src/PackageClassifier/PackageSourcesCache.cs]: `f => ...`

Comment (pranavkm) [inline src/PackageClassifier/PackageSourcesCache.cs]: `frameworkGroup`

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**